pavelib/assets.py

```diff
@@ -128,6 +128,8 @@ def compile_coffeescript(*files):
     ))
 
 
+@task
+@no_help
```

Comment:
I simply copied from the task above. Looks like `no_help` hides auxiliary tasks from the output of `paver help`.